sdk/cpp/core/src/common_utilities.cpp

```diff
@@ -0,0 +1,143 @@
+// YANG Development Kit
+// Copyright 2016 Cisco Systems. All rights reserved
+//
+////////////////////////////////////////////////////////////////
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+//////////////////////////////////////////////////////////////////
+
+#include "common_utilities.hpp"
+#include "entity_data_node_walker.hpp"
+#include "xml_subtree_codec.hpp"
+
+using namespace std;
+using namespace ydk;
```

Comment:
I think these functions need to be inside the `ydk` namespace as that is the convention being followed in the rest of the code. So this line can be removed and the below line can be used:
`namespace ydk {`